crates/consensus-logic/src/l1_handler.rs

```diff
@@ -187,6 +187,7 @@ pub fn verify_proof(checkpoint: &BatchCheckpoint, rollup_params: &RollupParams)
         RollupVerifyingKey::SP1VerifyingKey(vk) => {
             strata_sp1_adapter::verify_groth16(proof, vk.as_ref(), &public_params_raw)
         }
+        RollupVerifyingKey::NativeVerifyingKey(_) => Ok(()),
```

Comment:
For now, should we add a comment, considering that in native mode we perform public parameter assertions?